core/modules/layout/css/layout.admin.css

```diff
@@ -12,6 +12,41 @@
   color: #999;
 }
 
+@media (min-width: 768px) {
+  #layout-list span.priority-low {
+    display: inline;
+  }
+}
+
+#layout-list img {
+  display: none;
+  float: left; /* LTR */
+  margin-right: 10px;
+  max-width: 37.5px;
+  height: auto;
+}
+[dir="rtl"] #layout-list img {
+  float: right; /* LTR */
+  margin-left: 10px;
+}
+@media (min-width: 768px) {
+  #layout-list img {
+    display: inline;
+  }
+}
+
+#layout-list .layout-conditions-list ul {
+  margin-left: 0;
+}
+
+#layout-list .item-list ul li {
+  list-style-type: square;
```

Comment:
I hadnt considered the official style guide, just thought square bullets matched the template icons better. Will change them back.